README.md

```diff
@@ -17,3 +17,14 @@ The project structure is based on [`drupal-composer/drupal-project`](https://git
 	2. Run `docker-compose build`
 	3. Run `docker-compose up`
 3. Profit!
+
+## Client generation
+
+To generate the client code run the following command:
```

Comment:
Does the reader know the entire context so we can assume that they know what "the client" is?